go.mod

```diff
@@ -141,3 +141,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	libvirt.org/go/libvirt v1.11006.0 // indirect
 )
+
+replace github.com/osbuild/images => github.com/croissanne/images v0.0.0-20251023114920-81eefc443b5e
```

Comment:
...

Reply:
As this is still draft, this is just to show what it would look like with https://github.com/osbuild/images/pull/1966. Not actually intending to merge this.